_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,425 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $helper-text;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(small);
+        }
+
+        .helper-text { // Generally used for radios and checkboxes
+            margin-top: spacing-vertical(xx-small);
+            margin-left: 24px;
+        }
+    }
+}
+
+.form-label {
```

Comment:
Since `label` by itself is kinda generic, that it might be smart to define these labels as those that are in forms as opposed to say, navigation labels, video labels, bookmark labels, etc. What do you think?